api/src/main/resources/client_api.yaml

```diff
@@ -0,0 +1,139 @@
+# Endpoints in this API are intended to be used by researchers in notebooks; make sure they
```

Comment:
opt: maybe call this "base.yaml" or similar? Could make the relationship more obvious. Also nitpicking on naming client_api feels redundant to me (though I understand the intent); i.e. interaction with a client is implicit in the definition of "API".

Reply:
I don't love "base.yaml"... this is itself a full API specification.

Maybe "external_api.yaml"?

Reply:
Or "external_developer_api.yaml"?

Reply:
Eh, I don't like of these better than client_api, maybe just leave it for now. Other naming ideas include "core". It feels like there's some asymmetry with this approach, like maybe be need more than 2 yamls eventually, but fine to punt (e.g. base.yaml, workspaces.yaml, workbench.yaml, where the client lib would build base+workspaces)